src/components/Card.jsx

```diff
@@ -0,0 +1,28 @@
+import { Link } from 'gatsby'
```

Comment:
Looks like this is code from the other PR. There's a couple things we can do to fix this.
1. We can revert this back to what it was before. You should be able to just copy and paste from the current code base. 
2. We can make sure to merge your other PR first then see what merge conflicts happen or see if this just resolves itself. 
Either way let's make sure these changes aren't on this PR just for consistency's sake.